package.json

```diff
@@ -9,6 +9,7 @@
   },
   "devDependencies": {
     "aws-sdk": "2.595.0",
+    "aws-sdk-mock": "^5.1.0",
```

Comment:
```suggestion
    "aws-sdk-mock": "5.1.0",
```

Reply:
I propose using a fixed dependency version here, as a practice, to avoid problems with untested dependencies.